README.md

```diff
@@ -284,6 +284,12 @@ A number of options are available to allow you to customise the SDK:
   smsNumberCountryCode: 'US'
   ```
 
+- **`applicantDetails {Object} optional`**
+  Some applicant details can be specified ahead of time, so that the applicant doesn't need to fill them in themselves.
+
+  The following details can be used by the SDK:
+    - `smsNumber` (optional) : The applicant's mobile number, which can be used for sending any SMS messages to the applicant
```

Comment:
Can we give more details of what format we expect? i.e. country code or not?

Reply:
Good point, with an area-code is preferable so the user doesn't need to add it themselves afterwards
Will add

Reply:
Updated this now